setup.py

```diff
@@ -164,9 +169,34 @@ def run(self):
 with open('README.md') as fp:
   _LONG_DESCRIPTION = fp.read()
 
+package_name = package_config.PACKAGE_NAME
+install_requires = dependencies.make_required_install_packages()
+
+if package_config.PACKAGE_NAME == 'tfx-dev':
+  # Monolithic development package with the entirety of `tfx.*` and the full
+  # set of dependencies. Functionally equivalent to the union of the "tfx" and
+  # "tfx-pipeline-sdk" packages.
+  install_requires = dependencies.make_required_install_packages()
+  packages = find_namespace_packages(include=['tfx.*'])
+elif package_config.PACKAGE_NAME == 'tfx-pipelines-sdk':
+  # Core framework-neutral TFX pipeline SDK, without dependency on component-
+  # specific packages like "tensorflow" and "apache-beam".
+  install_requires = dependencies.make_pipeline_sdk_required_install_packages()
+  packages = find_namespace_packages(include=['tfx.dsl', 'tfx.proto'])
```

Comment:
Should this be an identiccal list with exclude on line 194 below? If so, move it to a local variable?

Reply:
Done.